style.css

```diff
@@ -60,328 +67,425 @@ body {
   font-weight: 500;
 }
 
+#navbar .web {
+  display: none;
+}
+
+/* Headline section */
+
 .presentation {
-  background-image: url(images/header.png);
+  width: 100%;
+  height: calc(100vh - 48px);
+  padding: 114px 24px;
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+  background-color: #fff;
+  background-image: url(images/header-mobile.png);
   background-position: center;
   background-repeat: no-repeat;
-  background-size: cover;
+  background-size: 100% 100%;
   border-bottom-left-radius: 100px;
-  flex: 1;
-  display: flex;
-  flex-direction: column;
+}
+
+.presentation h1 {
+  font-size: 40px;
+  font-weight: 700;
+  font-style: normal;
+  line-height: 52px;
+  letter-spacing: 0.37px;
+  color: #172b4d;
+}
+
+.presentation p {
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  color: #344563;
+}
+
+/* Works section */
+
+#works {
+  display: grid;
+  padding: 0 24px;
+  row-gap: 88px;
+  margin: 88px 0;
   justify-content: center;
-  align-items: center;
-  gap: 10px;
-  background-color: #fff;
-  width: 100%;
-  height: 100vh;
 }
 
-.content-padding {
-  padding: 10px 12px;
-  text-align: center;
+.card {
+  grid-column: 2;
+  background-color: white;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  padding: 16px;
+  gap: 12px;
 }
 
-#menu {
-  display: none;
+.card img.snapshot {
+  border-radius: 8px;
+  align-self: center;
 }
 
-/* content */
+.left-block {
+  display: flex;
+  flex-flow: column;
+  gap: 20px;
```

Comment:
- [ ] Hey @wassimchakib, you did a great job on this project. 👏 However, to make it look nicer, I would suggest you to increase a bit the padding-left of the left block. 
Like this 👇️
```.left-block {     padding-right: 129px; }```